styles/components/icons/mapping.less

```diff
@@ -124,6 +122,20 @@
 // ELM
 .icon-set(".elm", "elm", @blue);
 
+// EXCEL
+.icon-set(".xlsx", "xls", @green);
+.icon-set(".xlsm", "xls", @green);
+.icon-set(".xlsb", "xls", @green);
+.icon-set(".xltx", "xls", @green);
+.icon-set(".xltm", "xls", @green);
+.icon-set(".xls", "xls", @green);
+.icon-set(".xlt", "xls", @green);
+.icon-set(".xls", "xls", @green);
+.icon-set(".xlam", "xls", @green);
+.icon-set(".xla", "xls", @green);
+.icon-set(".xlw", "xls", @green);
+.icon-set(".xlr", "xls", @green);
```

Comment:
The .xlw and .xlr format are depreciated and rarely used, so they should probably be ommitted.

Reply:
Most of these formats are rarely used, why would that be a reason to not give them an icon?

Reply:
The main reason to remove them imo is still that they are depreciated. You can't even save to those 2 formats with the modern version of Excel unlike for all the others.

![image](https://github.com/user-attachments/assets/c1771d3d-0dc0-448a-92b2-a6df656b0ad4)

The fact that these 2 are extremely rare is just the cherry on top.